grpc-proto/src/test/java/io/stargate/grpc/TmpCqlExamples.java

```diff
@@ -0,0 +1,67 @@
+package io.stargate.grpc;
+
+import static io.stargate.grpc.CqlHelper.newStatement;
+import static io.stargate.grpc.CqlHelper.selectColumn;
+import static io.stargate.grpc.CqlHelper.selectWriteTime;
+import static io.stargate.grpc.CqlHelper.termOf;
+import static io.stargate.grpc.CqlHelper.whereColumn;
+import static io.stargate.grpc.CqlHelper.whereToken;
+import static stargate.Cql.Relation.Operator.EQ;
+
+import stargate.Cql.Relation.Operator;
+import stargate.Cql.Select;
+import stargate.Cql.Statement;
+
+public class TmpCqlExamples {
+
+  public static void main(String[] args) {
+    Statement statement;
+
+    // SELECT * FROM ks.tbl
+    statement = newStatement(Select.newBuilder().setKeyspaceName("ks").setTableName("tbl"));
+
+    // SELECT c1, c2 FROM ks.tbl
+    statement =
+        newStatement(
```

Comment:
The builder looks elegant, however, I think that we should discuss more:
> possibly more complicated for users not familiar with CQL.

I think it contradicts the direction that we wanted to achieve, that is abstracting away CQL from end-users as much as possible and not tight couple our APIs to the Cassandra backend.

Reply:
I am not sure I follow (or maybe I am missing context) -- this would not add (or reduce) need for CQL knowledge but offer another method of building CQL, mostly to be used by developers who build new APIs.

Reply:
In this approach, the caller application needs to know more about CQL grammar than just passing a plain CQL. The builder code will be used in more places in the caller code. It may possibly add some more tight coupling to Cassandra (as we may propagate it in the builder). If we would need to switch the backed from C* to other (that was one of the ideas of Stargate) it may be harder to map this new structure to a query, than just allowing user to pass CQL.

Reply:
Ok that part makes sense wrt knowledge. But one thing to note is that this is not an end-user API -- end users would use APIs, and APIs (and developers of APIs) would use structured approach. So abstraction would be at Document API or GraphQL or REST level.

I agree it is worth discussing at any rate.